docs/operations-guide/environment-variables.md

```diff
@@ -1,61 +1,810 @@
+
+#### `MB_QUERY_CACHING_MAX_TTL`
+
+Type: integer<br>
+Default: `8640000`
+
+The absolute maximum time to keep any cached query results, in seconds. The default value is 100 days in seconds.
+
+#### `MB_QUERY_CACHING_MIN_TTL`
+
+Type: integer<br>
+Default: `60`
+
+Metabase will cache all saved questions with an average query execution time longer than this many seconds.
+
+#### `MB_QUERY_CACHING_TTL_RATIO`
+
+Type: integer<br>
+Default: `10`
+
+To determine how long each saved question's cached result should stick around, we take the query's average execution time and multiply that by whatever you input here. So if a query takes on average 2 minutes to run, and you input 10 for your multiplier, its cache entry will persist for 20 minutes.
+
+#### `MB_REPORT_TIMEZONE`
+
+Type: string<br>
+Default: `null`
+
+Connection timezone to use when executing queries. Defaults to system timezone.
+
+#### `MB_SESSION_COOKIES`
+
+Type: boolean<br>
+Default: `null`
+
+When set to `true`, the user login sessions will not expire until the browser is closed. When not set, or set to `false`, the user login sessions will expire after the amount of seconds defined in [MAX_SESSION_AGE](#MAX_SESSION_AGE) (by default 2 weeks).
+
+#### `MB_SETUP_TOKEN`
+
+Type: string<br>
+Default: `null`
+
+An UUID token used to signify that an instance has permissions to create the initial User. This is created upon the first launch of Metabase, by the first instance; once used, it is cleared out, never to be used again.
+
+#### `MB_SHOW_HOMEPAGE_DATA`
+
+Type: boolean<br>
+Default: `null`
+
+Hide the “Our data" section from the homepage by setting it to `false`. Show the section with `true`, in case it was manually removed.
+
+#### `MB_SHOW_HOMEPAGE_XRAYS`
+
+Type: boolean<br>
+Default: `null`
+
+Hide the X-rays section from the homepage by setting it to `false`. Show the section with `true`, in case it was manually removed.
+
+#### `MB_SITE_LOCALE`
+
+Type: string<br>
+Default: `"en"`
+
+The default language for this Metabase instance. This only applies to emails, Pulses, etc. Users' browsers will specify the language used in the user interface.
+
+#### `MB_SITE_NAME`
+
+Type: string<br>
+Default: `"Metabase"`
+
+The name used for this instance of Metabase.
+
+#### `MB_SITE_URL`
+
+Type: string<br>
+Default: `null`
+
+The base URL where users access Metabase, e.g. `https://metabase.example.com` or `https://example.com/metabase`.
+
+#### `MB_SITE_UUID`
+
+Type: string<br>
+Default: `null`
+
+An UUID token used for this instance of Metabase.
+
+#### `MB_SLACK_TOKEN`
+
+Type: string<br>
+Default: `null`
+
+Slack API bearer token obtained from https://api.slack.com/web#authentication
+
+#### `MB_SOURCE_ADDRESS_HEADER`
+
+Type: string<br>
+Default: `X-Forwarded-For`
+
+Identify the source of HTTP requests by this headers value, instead of its remote address. Related to [MB_DISABLE_SESSION_THROTTLE](#MB_DISABLE_SESSION_THROTTLE)
```

Comment:
```suggestion
Identify the source of HTTP requests by this header's value, instead of its remote address. Related to [MB_DISABLE_SESSION_THROTTLE](#MB_DISABLE_SESSION_THROTTLE).
```